tests/ui/hashmap/hashmap-memory.rs

```diff
@@ -1,3 +1,4 @@
+//@ edition:2015
```

Comment:
Remark: there's a `"../tests/run-pass/hashmap-memory.rs"` string below, for a moment I thought this test was trying to load a no-longer existent file, but it turns out it's just some placeholder string, I think?